lib/libvfio-user.c

```diff
@@ -1113,65 +1121,90 @@ handle_device_feature(vfu_ctx_t *vfu_ctx, vfu_msg_t *msg)
     uint32_t supported_ops = device_feature_flags_supported(vfu_ctx, feature);
 
     if ((req->flags & supported_ops) != operations || supported_ops == 0) {
+        vfu_log(vfu_ctx, LOG_ERR, "unsupported operation(s)");
```

Comment:
Also log which operations for debugability.

Reply:
I'll log `req->flags` because it includes the feature and operations.